x-pack/plugins/cloud_security_posture/public/common/utils/get_vulnerability_colors.ts

```diff
@@ -8,15 +8,30 @@
 import { euiThemeVars } from '@kbn/ui-theme';
 
 export const getCvsScoreColor = (score: number): string | undefined => {
-  if (score >= 0.1 && score <= 3.9) {
+  if (score > 0 && score <= 4) {
```

Comment:
nit: we can give color to 0 or minus scores as low priority
it would make this function much more clear

as for undefined, we should check validity outside of the get score color IMO